test/test_axes.jl

```diff
@@ -63,3 +63,28 @@ end
     @test twpl[:top_margin] == 2Plots.cm
     @test twpl[:bottom_margin] == 2Plots.cm
 end
+
+@testset "aliases" begin
+    p = plot(1:2, xl = "x label")
+    @test p[1][:xaxis][:guide] === "x label"
+    p = plot(1:2, xrange = (0, 3))
+    @test xlims(p) === (0, 3)
+    p = plot(1:2, xtick = [1.25, 1.5, 1.75])
+    @test p[1][:xaxis][:ticks] == [1.25, 1.5, 1.75]
+    p = plot(1:2, xlabelfontsize = 4)
+    @test p[1][:xaxis][:guidefontsize] == 4
+    p = plot(1:2, gridalpha = .07)
+    @test p[1][:xaxis][:gridalpha] ≈ .07
```

Comment:
Does `gridalpha` only set the value for the x-axis or all grids?

Reply:
Sorry, it was an un-voluntary change, I used `xgα = ...` in my initial test. Fixed.